docs/content/docs/dev/table/sql/create.md

```diff
@@ -197,6 +197,19 @@ CREATE TABLE [IF NOT EXISTS] [catalog_name.][db_name.]table_name
 The statement above creates a table with the given name. If a table with the same name already exists
 in the catalog, an exception is thrown.
 
+For information about CREATE MATERIALIZED TABLE, please refer to [Materialized Table Statements]({{< ref "docs/dev/table/materialized-table/statements" >}}).
+
+**TEMPORARY**
+
+Temporary tables are always stored in memory and only exist for the duration of
+the Flink session they are created within.
+
+Check out more details at [Temporary vs Permanent tables]({{< ref "docs/dev/table/common" >}}#temporary-vs-permanent-tables).
+
+**IF NOT EXISTS**
```

Comment:
I suggest removing the IF NOT EXISTS. As this is not always strictly true. See https://nightlies.apache.org/flink/flink-docs-master/api/java/org/apache/flink/table/api/TableEnvironment.html#createTemporaryView(java.lang.String,org.apache.flink.table.api.Table) javadoc  `If a temporary object in a given path exists, the permanent one will be inaccessible in the current session. `

Reply:
ok, i will fix it at this weekend